vocabularies/voc4cat/0000302.ttl

```diff
@@ -0,0 +1,17 @@
+PREFIX dcterms: <http://purl.org/dc/terms/>
+PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
+PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
+
+<https://w3id.org/nfdi4cat/voc4cat_0000302>
+    a skos:Concept ;
+    dcterms:identifier "voc4cat_0000302"^^xsd:token ;
+    skos:altLabel
+        "isomerisation reaction"@en ,
+        "isomerization"@en ,
+        "isomerization reaction"@en ;
+    skos:broader <https://w3id.org/nfdi4cat/voc4cat_0000185> ;
```

Comment:
Should this also have skos:broader voc4cat:0000183 (event) like other types of reactions?